packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -281,14 +281,21 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
     schemaComposer.Query.addFields(queryFields)
   }
 
-  protected createStandardTypes(schemaComposer: SchemaComposer<any>) {
-    schemaComposer.add(IDScalarInputType)
-    schemaComposer.add(StringScalarInputType)
-    schemaComposer.add(IntScalarInputType)
-    schemaComposer.add(FloatScalarInputType)
-    schemaComposer.add(BooleanScalarInputType)
-    schemaComposer.add(SortDirectionEnum)
-    schemaComposer.add(PageRequest)
+  private createStandardInputTypes(schemaComposer: SchemaComposer<any>, schema: GraphQLSchema) {
+    schemaComposer.add(PageRequest);
+    schemaComposer.add(IDScalarInputType);
+    schemaComposer.add(SortDirectionEnum);
+    schemaComposer.add(StringScalarInputType);
+    schemaComposer.add(BooleanScalarInputType);
+    schemaComposer.add(createInputTypeForCustomScalar(GraphQLInt));
+    schemaComposer.add(createInputTypeForCustomScalar(GraphQLFloat));
+    
+    const schemaTypes = Object.values(schema.getTypeMap());
```

Comment:
Kinda liked previous helper as it could become an core thing :D

Reply:
I thought I could remove it for now since it meant iterating twice on.

Reply:
You can getTypeMap on schemaComposer, so you don't need to pass schemaComposer and schema.

Reply:
Yeap, that's the initial idea we had with @wtrocki but could not find a simple way to get them from schemaComposer. Suggestions welcomed.

Reply:
I tried this out and it seems to work:

```ts
private createStandardInputTypes(schemaComposer: SchemaComposer<any>, schema: GraphQLSchema) {
    schemaComposer.add(PageRequest);
    schemaComposer.add(IDScalarInputType);
    schemaComposer.add(SortDirectionEnum);
    schemaComposer.add(StringScalarInputType);
    schemaComposer.add(BooleanScalarInputType);
    schemaComposer.add(createInputTypeForCustomScalar(GraphQLInt));
    schemaComposer.add(createInputTypeForCustomScalar(GraphQLFloat));

    schemaComposer.forEach((tc: NamedTypeComposer<any>) => {
      const namedType = tc.getType()
      if (isScalarType(namedType) && !isSpecifiedScalarType(namedType)) {
        schemaComposer.add(createInputTypeForCustomScalar(namedType));
      }
    })
}
```

Reply:
Okay, let's use that version and use schemaComposer as single source of truth.